var/spack/repos/builtin/packages/turbovnc/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Turbovnc(CMakePackage):
+    """TurboVNC is a derivative of VNC (Virtual Network Computing)
+    that is tuned to provide peak performance for 3D and video workloads.
+    TurboVNC was originally a fork of TightVNC 1.3.x, on the surface,
+    the X server and Windows viewer still behave similarly to their parents."""
+
+    homepage = "http://www.turbovnc.org/"
+    url      = "http://downloads.sourceforge.net/project/turbovnc/2.0.1/turbovnc-2.0.1.tar.gz"
+
+    version('2.1.2', '9d04b26d40664d0d8072e8b1828f69c4')
+    version('2.1.1', 'b1b1537eb5f8e6bd90acfd853277a9cf')
+    version('2.1', '6748bb13647d318f0c932394f8298d10')
+    version('2.0.1', 'a279fdb9ac86a1ebe82f85ab68353dcc')
+    version('1.2.2', '556d97741199cf4e2f2d2b2fbee0069c')
+
+    variant('java',    default=False, description='Enable Java build')
+    variant('server',  default=True,  description='Enable server build')
+    variant('x11deps', default=True,  description='Depends x11 depends')
```

Comment:
Description doesn't make a ton of sense to me... Maybe rename variant to `x11` and say `Add X11 dependencies`